spec.html

```diff
@@ -36349,7 +36349,7 @@ <h1>JSON.stringify ( _value_ [ , _replacer_ [ , _space_ ] ] )</h1>
             1. Set _space_ to ? ToString(_space_).
         1. If Type(_space_) is Number, then
           1. Set _space_ to min(10, ! ToInteger(_space_)).
-          1. Let _gap_ be the String value containing _space_ occurrences of the code unit 0x0020 (SPACE). This will be the empty String if _space_ is less than 1.
+          1. If _space_ &le; 0, let _gap_ be the empty String; otherwise let _gap_ be the String value containing _space_ occurrences of the code unit 0x0020 (SPACE).
```

Comment:
```suggestion
          1. If _space_ &lt; 1, let _gap_ be the empty String; otherwise let _gap_ be the String value containing _space_ occurrences of the code unit 0x0020 (SPACE).
```

Reply:
a360b96